pkg/eventing/v1beta1/client.go

```diff
@@ -183,10 +183,10 @@ func (b *TriggerBuilder) Broker(broker string) *TriggerBuilder {
 // InjectBroker to add annotation to setup default broker
 func (b *TriggerBuilder) InjectBroker(inject bool) *TriggerBuilder {
 	if inject {
-		meta_v1.SetMetaDataAnnotation(&b.trigger.ObjectMeta, v1beta1.DeprecatedInjectionAnnotation, "enabled")
+		meta_v1.SetMetaDataAnnotation(&b.trigger.ObjectMeta, v1beta1.InjectionAnnotation, "enabled")
 	} else {
-		if meta_v1.HasAnnotation(b.trigger.ObjectMeta, v1beta1.DeprecatedInjectionAnnotation) {
-			delete(b.trigger.ObjectMeta.Annotations, v1beta1.DeprecatedInjectionAnnotation)
+		if meta_v1.HasAnnotation(b.trigger.ObjectMeta, v1beta1.InjectionAnnotation) {
```

Comment:
funny ping pong with the `Deprecated` Prefix. Really not a big fan of this way of deprecation that already breaks the API (so it's kinda funny to deprecate and break backwards compatibility at the same time :). I already mentioned that over at serving, so maybe that's the reason for reverting this renaming?

Reply:
how I understood the constant deprecation here is; there is a time frame (3 releases IIRC) the deprecated values are honored and during this the consumers are expected to move to newer constants. For client, we do it at the end of the time frame when the constant is no longer available in the vendored library. Beside, I think the respective controller was considering both deprecated and new values; but after a point it doesn't. So with this value, client should be able to work with earlier 3 releases, please correct me if I got this wrong.

Reply:
We shipped both `DeprecatedInjectionAnnotation` and `InjectionAnnotation`, you could have continued to use `InjectionAnnotation` and it would have worked.